src/components/MovieDetailsPage/DetailsComponents/ReleaseYear.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react';
+import { FontSize } from '../../../model/enums/FontSize';
+import { MovieInfoDetails } from '../../../types';
+import { AppText } from '../../shared';
+
+type ReleaseYearProps = {
+  chosenMovieDetails: MovieInfoDetails
+};
+
+export const ReleaseYear: FC<ReleaseYearProps> = (props) => {
+  const { chosenMovieDetails } = props;
+  const { year } = chosenMovieDetails;
+  return (
+    <AppText text={year.toString()} color="red" fontSize={FontSize.MEDIUM} />
```

Comment:
do we need to convert it to string?

Reply:
Added opportunity to add number without converting to string. Commit d0df8953a6498f884fc8f239cf86721b7a4c4d9a.